Adyen/Model/Payment.swift

```diff
@@ -31,4 +31,8 @@ public struct Payment {
         self.countryCode = unsafeCountryCode
     }
 
+    private enum CodingKeys: String, CodingKey {
+        case amount
+        case countryCode
+    }
```

Comment:
Thanks @sebastianvarela for your contribution, its appreciated! :-)

just a small comment, I think `CodingKeys` is redundant, since Swift compiler provide it automatically whenever all members are `Codable`, am I missing something?

Reply:
yes @mohammedDehairy, you are right! we removed... PR updated!